src/main/java/com/softserve/sprintone/ivan/TaskIvanSprintOne.java

```diff
@@ -0,0 +1,422 @@
+package com.softserve.sprintone.ivan;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+import static java.lang.Math.pow;
+import static java.lang.Math.random;
+
+/**
+ * @author Iván Caballero
+ */
+
+/**
+ * Task 13 - The code provided has a method plus100 which is supposed to return the number of the
+ * parameter incremented by 100. But it's not working properly.
+ * Constraints:
+ * - Do not modify the constructor
+ * - Do not modify the plus100 method
+ * - Do not modify the ONE_HUNDRED declaration
+ */
+class Dinglemouse {
```

Comment:
use a different class for this one, it is not a good practice to have more than one class in a single file.

Reply:
Ok, got it